ArduCopter/fence.cpp

```diff
@@ -81,7 +85,8 @@ void Copter::fence_check()
 
         LOGGER_WRITE_ERROR(LogErrorSubsystem::FAILSAFE_FENCE, LogErrorCode(new_breaches));
 
-    } else if (orig_breaches) {
+    } else if (orig_breaches && fence.get_breaches() == 0) {
+        GCS_SEND_TEXT(MAV_SEVERITY_NOTICE, "Fence breach cleared");
```

Comment:
I think its odd that this always print Fence breach cleared but only prints the breach if `!copter.ap.land_complete`. The result could be you get lots of breach cleared messages with no messages of the original breach.

Reply:
Yeah, let's see what @rmackay9 thinks

Reply:
Thanks, fixed